agent/fw_drupal8.c

```diff
@@ -410,28 +411,32 @@ NR_PHP_WRAPPER(nr_drupal94_invoke_all_with) {
 
   hook = nr_php_arg_get(1, NR_EXECUTE_ORIG_ARGS TSRMLS_CC);
   if (!nr_php_is_zval_non_empty_string(hook)) {
-      goto leave;
+    goto leave;
   }
 
   prev_hook = NRPRG(drupal_module_invoke_all_hook);
   prev_hook_len = NRPRG(drupal_module_invoke_all_hook_len);
   NRPRG(drupal_module_invoke_all_hook)
       = nr_strndup(Z_STRVAL_P(hook), Z_STRLEN_P(hook));
   NRPRG(drupal_module_invoke_all_hook_len) = Z_STRLEN_P(hook);
-
+  NRPRG(check_cufa) = true;
   callback = nr_php_arg_get(2, NR_EXECUTE_ORIG_ARGS TSRMLS_CC);
   /* This instrumentation will fail if callback has already been wrapped
    * with a special instrumentation callback in a different context.
    * In this scenario, we will be unable to instrument hooks and modules for
    * this particular call */
-  nr_php_wrap_generic_callable(callback, nr_drupal94_invoke_all_with_callback TSRMLS_CC);
+  nr_php_wrap_generic_callable(callback,
+                               nr_drupal94_invoke_all_with_callback TSRMLS_CC);
```

Comment:
```suggestion
                               nr_drupal94_invoke_all_with_callback);
```

Reply:
Not related to PR after reverting format changes.